dev/NavigationView/NavigationView.cpp

```diff
@@ -365,14 +365,23 @@ void NavigationView::OnApplyTemplate()
     if (SharedHelpers::IsThemeShadowAvailable())
     {
 #ifdef USE_INSIDER_SDK
-        if (auto splitView = m_rootSplitView.get())
+        // Shadow will get clipped if casting on the splitView.Content directly
+        // Creating a canvas with negative margins as receiver to allow shadow to be drawn outside the content grid 
+        winrt::Canvas shadowReceiver;
+        winrt::Thickness shadowReceiverMargin = { -OpenPaneLength(), -c_paneElevationTranslationZ, -c_paneElevationTranslationZ, -c_paneElevationTranslationZ };
+        shadowReceiver.Margin(shadowReceiverMargin);
+
+        if (auto contentGrid = GetTemplateChildT<winrt::Grid>(c_contentGridName, controlProtected))
         {
-            if (auto contentRoot = splitView.Content())
+            contentGrid.SetRowSpan(shadowReceiver, contentGrid.RowDefinitions().Size());
+            contentGrid.Children().Append(shadowReceiver);
+
+            winrt::ThemeShadow shadow;
+            shadow.Receivers().Append(shadowReceiver);
+            if (auto splitView = m_rootSplitView.get())
             {
                 if (auto paneRoot = splitView.Pane())
                 {
-                    winrt::ThemeShadow shadow;
-                    shadow.Receivers().Append(contentRoot);
                     if (winrt::IUIElement10 paneRoot_uiElement10 = paneRoot)
```

Comment:
>winrt::IUIElement10 paneRoot_uiElement10 = paneRoot [](start = 24, length = 51)

It's now a bug, but I prefer to
auto paneRoot_uiElement10 = paneRoot.try_as<winrt::IUIElement10 >()